core/app/models/spree/wallet/add_after_order_complete.rb

```diff
@@ -0,0 +1,25 @@
+class Spree::Wallet::AddAfterOrderComplete
+  def initialize(order)
+    @order = order
+  end
+
+  # AddAfterOrderComplete is called after an order transitions to complete. It
+  # is responsible for saving payment sources in the user's "wallet" for future
+  # use.
+  def add_to_wallet
```

Comment:
Given this comment (https://github.com/solidusio/solidus/pull/1086/files#diff-a9fae1594c22d1ec005ac8501d80d7afR356) and the importance of documenting these interfaces, could we add some more YARD docs to this method?